app/views/project_comments/_comment.html.erb

```diff
@@ -24,7 +24,7 @@
             </a>
           </li>
         <% end %>
-        <% if can_delete_project_comment(comment) %>
```

Comment:
Could you merge these 2 checks here into one?

Reply:
Sure buddy!

Reply:
Actually I've just deleted both, since the parent permission check is the same.